test/yaml.go

```diff
@@ -39,6 +39,8 @@ const (
 	sonataFlowPlatformForOpenshift            = "sonataflow.org_v1alpha08_sonataflowplatform_openshift.yaml"
 	sonataFlowBuilderConfig                   = "sonataflow-operator-builder-config_v1_configmap.yaml"
 
+	BuilderDockerfile = "builder_dockerfile.yaml"
+
```

Comment:
Is this used anywhere?

Reply:
No I refactored the test to use the builder/Dockerfile, I'll remove it